src/steamship/base/tasks.py

```diff
@@ -31,7 +30,7 @@ class ListTaskCommentRequest(Request):
 
 
 class TaskComment(CamelModel):
-    client: Any = None
+    client: Any = Field(None, exclude=True)
```

Comment:
Potentially a cleaner solution to remove client from serialisation (e.g. .dict()). 

This will become way cleaner once all our objects use a base class. But for now I do think it's cleaner since the user can understand why client will be excluded from dict by looking at the data model (vs nested in the lambda handler).

Reply:
WDYT?

Reply:
I don't have a strong opinion personally, though I do agree this is more explicit than being clever and "magical" down at the serialization level. Happy to run with this approach if it works!